libsolidity/interface/Natspec.cpp

```diff
@@ -37,6 +37,9 @@ Json::Value Natspec::userDocumentation(ContractDefinition const& _contractDef)
 	Json::Value doc;
 	Json::Value methods(Json::objectValue);
 
+	doc["version"] = Json::Value(static_cast<Json::Value::UInt>(NATSPEC_VERSION));
```

Comment:
If you make `NATSPEC_VERSION` as `unsigned int`, then we can avoid this static cast.